src/CategoryCard.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Card, CardHeader, IconButton, Avatar } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(1),
+    maxWidth: 500,
+    margin: '10px auto',
+  },
+}));
+
+type Props = {
```

Comment:
Maybe `CardProps` would be a better name, what do you think?

Reply:
I haven't got any strong opinion about this. My way of thinking was that these props are already in the category card file and the prefix is self-implying. If it's kind of good practice or convention, let me know, I'll gladly change it.

Reply:
I think it's better to give names that are self-explaining. Than looking on the code you know right away, what is this. Otherwise you would have to think in which component you actually are. I wrote you that, because that's exaclty what I did. The project is really small but I came here and I was thinking for few seconds where am I :P

Reply:
Ok, thanks. It makes sense. I'll rename it ✌️

Reply:
Done ✅